extensions/github-repository-search/src/useRepositories.ts

```diff
@@ -35,15 +36,45 @@ query SearchRepositories($searchText: String!) {
   }
 }`;
 
-export function useRepositories(searchText: string | undefined) {
+function buildSearchText(baseSearchText?: string): string | undefined {
+  if (!baseSearchText) {
+    return;
+  }
+
+  const searchTextParts: string[] = [];
+  if (preferenceValues.searchInOrgs) {
+    const orgs = preferenceValues.searchInOrgs
+      .split(",")
+      .map((org) => org.trim())
+      .filter((org) => org.length > 0);
+    if (orgs.length > 0) {
+      const orgsSearchText = orgs.map((org) => `user:${org}`).join(" ");
```

Comment:
Does this actually work for organizations like `raycast`? The docs mention that there is a `user:` and `org:` filter.

Reply:
Yes, you can see that in the second video of the PR description, where `raycast,@me` is set and `includeForks` is enabled. There it returns results for both raycast and my personal repos.

From what I understood from testing, filtering for `user:X` and `org:X` does behave identical with the exception that `org:@me` does not work.

Reply:
Ok, then lets leave as is. If people enter `user:thomaspaulmann` and `org:raycast` as additional filters, it should still work.